website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -41,7 +41,9 @@ Placeholder text can be added to the trigger to provide the user additional cont
 
 **Filled**
 
-When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction, as needed. It is important to note that the selected values in the trigger are displayed in the order chosen by the user.
+When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction, as needed. It is important to note that the selected values in the trigger are displayed in the order chosen by the user. 
```

Comment:
```suggestion
When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction as needed. It is important to note that the selected values in the trigger are displayed in the order chosen by the user. 
```

This seems like a leftover from the previous content, but I'm not sure if this comma is needed.